app/assets/src/components/views/amr_heatmap/AMRHeatmapControls.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Divider } from "~/components/layout";
+import { Dropdown } from "~ui/controls/dropdowns";
+
+import cs from "./amr_heatmap_view.scss";
+
+export default class AMRHeatmapControls extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  generateChangeFunction(filter) {
+    const changeFunction = option => {
+      if (option === this.props.selectedOptions[filter]) {
+        return;
+      }
+      this.props.onSelectedOptionsChange({ [filter]: option });
+    };
+    return changeFunction;
+  }
+
+  renderFilterDropdowns() {
+    const filtersList = [];
+    this.props.filters.forEach((_, filter) => {
+      filtersList.push(
+        <div className="col s3">
+          <Dropdown
+            fluid
+            rounded
+            options={this.props.filters.get(filter).options}
+            onChange={this.generateChangeFunction(filter)}
+            value={this.props.selectedOptions[filter]}
+            label={this.props.filters.get(filter).label}
+            disabled={!this.props.data}
+          />
+        </div>
+      );
+    });
+    return filtersList;
+  }
+
+  render() {
+    return (
+      <div className={cs.menu}>
+        <Divider />
+        <div className={`${cs.filterRow} row`}>
```

Comment:
you should actually use `classnames` instead of interpolation for this one (esp if you ever need conditional classnames)

`import cx from "classnames";`
and
`cx(cs.filterRow, row)`